manpage/wf-recorder.1

```diff
@@ -206,7 +201,7 @@ data for
 .Ql vaapi
 input but really only support yuv planar formats.
 In this case, use the
-.Fl t , -force-yuv
+.Fl x yuv420p , -pixel-format yuv420p
```

Comment:
Something is wrong here, after rendering it looks like this:

```
In this case, use the -x -yuv420p,
     --pixel-format -yuv420p option in addition to the ‘vaapi’ options to convert the data to yuv planar data before sending it to the GPU.
```

Reply:
Hm it renders fine for me but I changed it anyway since it was redundant

Reply:
It is better now, but I still get `use the -x -yuv420p option` with both neovim and the default manpager.

Reply:
In fact, some other options (which you have not changed at all) are also broken in the manpage, I wonder what is going on.

Reply:
Aha, so you need `.Fl x Ar yuv420p`

Reply:
I changed everything that seemed like obvious inconsistencies to me